app/vue/src/client/preview/types.ts

```diff
@@ -0,0 +1,33 @@
+// TODO, 'any' should be what is actually expected from a storyFn
+
+export interface ShowErrorArgs {
+  title: string;
+  description: string;
+}
+
+export interface RenderMainArgs {
+  storyFn: () => any;
+  selectedKind: string;
+  selectedStory: string;
+  showMain: () => void;
+  showError: (args: ShowErrorArgs) => void;
+  showException: (...args: any[]) => void;
+  forceRender: boolean;
+}
+
+// TODO: some vue expert needs to look at this
+export type StoryFnVueReturnType = any;
+
+export interface ICollection {
```

Comment:
why sometimes prefix with `I` but sometimes not?  Can we have some convention here (I know there is a ESLint-TypeScript rule for that out of box...)?

Reply:
I personally prefer the naming convention without the I.

Reply:
Having some more consistency wouldn't hurt, but it's not a really big deal IMHO.

Reply:
I guess due to the fact that type scoped variables can be imported/exported together with the normal lexical scoped variables, so it is a bit hard to read/review without using IDE. So it makes sense to have prefix to give hints in such cases.  I'm also preferring without `I` though.

While it is true that it just a matter of code cosmetic and bike-shading, but it is actually quite important to maintain some level of consistency so more people can involve and make contribution easier, IMHO.